mbs_results/data_cleaning.py

```diff
@@ -177,3 +180,96 @@ def load_manual_constructions(
     return df.merge(
         manual_constructions, on=[reference, period], how="outer", suffixes=("", "_man")
     )
+
+
+def run_live_or_frozen(
+    df: pd.DataFrame,
+    target: str or list[str],
+    error_marker: str,
+    state: str = "live",
+    error_values: List[str] = ["E", "W"],
+) -> pd.DataFrame:
+
+    """
+    For frozen, therefore target values are converted to null, hence responses
+    in error are treated as non-response.
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        Original dataframe.
+    target : str or list[str]
+        Column(s) to treat as non-response.
+    error_marker : str
+        Column name with error values.
+    state : str, optional
+        Function config parameter. The default is "live".
+    error_values : list[str], optional
+        Values to ignore. The default is ['E', 'W'].
+
+    Returns
+    -------
+    Original dataframe.
+
+    """
+
+    # TODO: raise error if state is not frozen or live
```

Comment:
What is supposed to happen if the state is not live or frozen. Could this be added now?